bin/trace

```diff
@@ -305,10 +307,18 @@ sub rewrite_func ($$$$$) {
     $trace = "H5TRACE0(\"$rettype\", \"\");\n";
     $argtrace = "H5ARG_TRACE0(\"\")";
   } else {
-    # Split arguments.  First convert `/*in,out*/' to get rid of the
-    # comma and remove lines beginning with a '#', then split the arguments
-    # on commas.
-    $args =~ s/(\/\*\s*in),\s*(out\s*\*\/)/$1_$2/g;     # Get rid of comma in 'in,out'
+    # Split arguments
+    #
+    # First remove:
+    #   * /*in*/, /*out*/, /*in_out*/, and /*in,out*/ comments
+    #   * preprocessor lines that start with #
+    #   * H5FL_TRACKING_PARAMS macros (free list code only)
+    #
+    # then split the function arguments on commas
+    $args =~ s/\/\*\s*in\s*\*\///g;  # Get rid of /*in*/
+    $args =~ s/\/\*\s*out\s*\*\///g; # Get rid of /*out*/
+    $args =~ s/\/\*\s*in,\s*out\s*\*\///g; # Get rid of /*in,out*/
+    $args =~ s/\/\*\s*in_out\s*\*\///g; # Get rid of /*in,out*/
```

Comment:
Very minor nitpick that doesn't really matter: `in,out` -> `in_out` in the comment

Reply:
No problem. Will fix.